modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -12039,12 +12037,10 @@
                                   "phoneNumber": {
                                     "description": "Phone number.",
                                     "type": "string",
-                                    "pattern": "^[0-9]{1,14}$",
-                                    "example": "555-5555"
-                                  },
-                                  "phoneNumberExt": {
-                                    "type": "string",
-                                    "pattern": "^[a-zA-Z0-9]{1,10}$"
+                                    "pattern": "^\\d(?:[- ]?\\d){0,13}$",
+                                    "example": "555-5555",
+                                    "minLength": 1,
+                                    "maxLength": 14
```

Comment:
I swear this is the last one! Since we allow up to 14 digits with various dashes and spaces in-between, a maxLength of 14 is too small to account for that. I don't think going deep into the math of exactly how many characters is allowed by the regex is necessary - 32 should be plenty of room.